src/liquid/components/ld-tooltip/readme.md

```diff
@@ -0,0 +1,251 @@
+---
+eleventyNavigation:
+  key: Tooltip
+  parent: Components
+layout: layout.njk
+title: Tooltip
+permalink: components/ld-tooltip/
+---
+
+# ld-tooltip
+
+Tooltips provide additional information, mostly short paragraphs, and can be placed besides all sorts of interface elements.
+
+---
+
+## Default
+
+{% example %}
+<ld-tooltip>
+  <ld-heading level="4" style="margin-bottom: 10px">Headline</ld-heading>
```

Comment:
This comment applies to all examples below: Can we use relative units in the examples (`rem`)? Using relative units is good practice in regards to a11y and we should lead by example.